docs/about/why_neon.md

```diff
@@ -13,11 +13,11 @@ comments: na
 
 Neon EVM is a fully compatible Ethereum environment solution built on the Solana blockchain. The platform allows developers to deploy Ethereum dApps without undertaking any reconfiguration. 
 
-> No code changes required: Ethereum developers can build without Rust.
+> Minimal reconfiguration required: Ethereum developers can build without Rust.
 
 Neon EVM enables Solidity- and Vyper-based dApps to tap into Solana’s network benefits: low fees, high transaction speeds, and parallel transaction execution capabilities.
 
-From a developer’s perspective, this means creating Solana-compatible contracts using tools you’re familiar with from the Ethereum ecosystem. 
+From a developer’s perspective, this means creating Solana-compatible contracts using tools you’re familiar with from the Ethereum Ecosystem. 
 
 Neon EVM makes the key Ethereum dApp tools compatible with Solana, including Vyper, Solidity, [MetaMask](/docs/wallet/metamask_setup), [Hardhat](/docs/developing/deploy_facilities/using_hardhat), [Truffle](/docs/developing/deploy_facilities/using_truffle), and [Remix](/docs/developing/deploy_facilities/using_remix). The solution allows any Ethereum application to run on Solana without any changes to its codebase, this includes Uniswap, SushiSwap, 0x, and MakerDAO. 
```

Comment:
“The solution allows any Ethereum application to run on Solana **without any changes** to its codebase” pls change to “Minimal reconfiguration required”

Reply:
✅